test/functional/operation_example.test.js

```diff
@@ -3365,7 +3365,7 @@ describe('Operation Examples', function () {
    * @example-class Db
    * @example-method collection
    */
-  it('shouldCorrectlyAccessACollection', {
+  it.skip('shouldCorrectlyAccessACollection', {
```

Comment:
why is this test being skipped?

Reply:
They are using the deprecated form of the helper; I think they should probably just be deleted rather than rewritten since they are just providing examples, not coverage, and the helper is much more self-explanatory in its behavior now. Does that sound reasonable?

Reply:
I don't mind deleting, since we need to consolidate/reorganize our test suite anyway, as long as we do have the actual coverage from elsewhere